README.md

```diff
@@ -4,27 +4,43 @@ Automating MakePlayingCards's online ordering system.
 The below guide describes the procedure for setting up the web component. If you're here to download the clientside program, check the Releases tab.
 
 # Requirements
-`requirements.txt` for the web application and local tool combined exists in the repo as well.
+`requirements.txt` for the web application and local tool combined.
 
-## Web application:
-JS libraries:
-* `bootstrap@4.6.0`
-* `jquery-ui-touch-punch@0.2.3`
-* `slick.js@1.8.1`
-
-Other:
-* [Bootstrap Superhero](https://bootswatch.com/superhero/)
```

Comment:
These lines (9-16) shouldn't be deleted since they are still necessary dependencies. It would also be good to explain that the relevant CSS & JS files need to be brought into carpicker\static.

Also replace bootswatch link with the bootstrapper 4 version

Reply:
good catch - however, I believe PR #35 updates the project to source these files from CDN rather than requiring them to be downloaded & stored in `/static` - will aim to merge that PR first and this PR can update the readme with any changes afterwards